HTML/public/homepage.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Explore the Milky Way</title>
+    <link rel="stylesheet" href="/HTML/public/css/style.css" />
+    <link rel="icon" type="DLOR" href="/HTML/public/assets/homepage/DLOR_2023.svg" />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script>
+      tailwind.config = {
+        theme: {
+          extend: {
+            backgroundImage: {
+              "space-milkyway": "url('./assets/homepage/Main_Background.png')",
+            },
+            dropShadow: {
+              custom1: "0px 0px 6px rgba(180, 113, 7, 0.8)",
+            },
+            fontFamilly: {
+              arcade: ["Arcade Classic"],
```

Comment:
and in here, update your custom `fontFamily` with `VP Pixel` to

Reply:
done, aku lupa hehe